### PR TITLE
Cristian part3 Integration

### PR DESCRIPTION
**Pull request by snaket2003**
Integrating Part3

### CODE REVIEW

File: src/main/java/com/codingnomads/part_03/Exercise_05.java [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
-
-
+        if (userNum % 4 == 0 && userNum % 7 == 0) {
+            System.out.println("it is divisible by 4 and 7");
+        } // use the || operator to see if the user's number is divisible by 4 or 7
+        else if (userNum % 4 == 0 || userNum % 7 == 0) {
+            System.out.println("it is divisible by 4 or 7");
+        } // use the ^ operator to see if the user's number is divisible for 4 or 7 exclusively
+        else if (userNum % 4 == 0 ^ userNum % 7 == 0) {
+            System.out.println("it is divisible for 4 or 7 exclusively");
```
Comment (alpdurmaz) [inline src/main/java/com/codingnomads/part_03/Exercise_05.java]: last condition with ^ operator is not reachable in any case. You can check it with for loop and see.

File: src/main/java/com/codingnomads/part_03/Exercise_06.java [hunk 2]
```diff
@@ -4,30 +4,57 @@
...
         // Get the current hour
+        long currentHour = totalHours % 60;
 
         // Display results using a 12 hour clock, include AM or PM
 
+        String formattedTime = String.format("%02d", currentHour) + ":" + String.format("%02d", minuteWithinTheHour)
+                + ":" + String.format("%02d", secondWithinTheHour);
+
+        System.out.println(formattedTime);
+
```
Comment (alpdurmaz) [inline src/main/java/com/codingnomads/part_03/Exercise_06.java]: Your hour is 12 hours ahead from current time. You can check it again.

File: src/main/java/com/codingnomads/part_03/Exercise_05.java (hunk 1 above)
Comment (snaket2003) [reply src/main/java/com/codingnomads/part_03/Exercise_05.java]: Great catch - I have adjusted the 'else if' statements for 'if'

File: src/main/java/com/codingnomads/part_03/Exercise_06.java (hunk 2 above)
Comment (snaket2003) [reply src/main/java/com/codingnomads/part_03/Exercise_06.java]: I have adjusted for modulo 12 in order to fix the problem

**Pull request merged**